examples/notepad/index.html

```diff
@@ -1,33 +1,157 @@
 <!DOCTYPE HTML>
 <html lang="en">
-  <head>
```

Comment:
Can you have a quick look at what I did here?  I'm not completely done.  I wish it were shown a bit more like a "real" example...  just want to get your thoughts?

Reply:
Why did you unindent this? the `<head>` tag comes under the `<html>` tag, right?

Reply:
hmm, don't think it was intentional.  I will take a look

Reply:
on my screen it does not show as indented.  I will resolve this and see if it appears again.